packages/react-app/package.json

```diff
@@ -124,6 +124,7 @@
     "start": "cross-env HTTPS=true npm run sync-assets && react-app-rewired start",
     "start:local": "cross-env REACT_APP_CHAIN_ID=1337 yarn start",
     "start:rinkeby": "cross-env REACT_APP_CHAIN_ID=4 yarn start",
+    "start:rinkeby-staging": "cross-env REACT_APP_CHAIN_ID=4 REACT_APP_STAGING=true yarn start",
```

Comment:
To who want test it in vercel staging need run that.

Reply:
Nope vercel always run yarn start and rely on defaultChain and staging variables to decide wich network and environment